recipes/openblas/all/conanfile.py

```diff
@@ -28,7 +32,6 @@ class OpenblasConan(ConanFile):
         "use_thread": True,
         "dynamic_arch": False,
     }
-    generators = "cmake"
     short_paths = True
 
     @property
```

Comment:
`self._source_subfolder` and `self._build_subfolder` can be simply replaced with `self.source_folder` and `self.build_folder` or even omitted entirely in most cases with Conan 2.